### PR TITLE
Contact form pull request

### PR DESCRIPTION
**Pull request by codehass**
In this Contac form pull request : 
 - Add footer section HTML code
 - Add images for the footer section.
 - Add style for the footer section. 

- Add required to the input element.
- Add button: hover and button: active.

Thank you for reviewing my pull request.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -193,6 +193,59 @@ <h3>Skills</h3>
...
+                id="name"
+                name="username"
+                maxlength="30"
+                placeholder="Full name"
+                required
+              />
+            </li>
+            <li>
+              <label for="">Your email</label>
+              <input type="email" placeholder="Your email" />
```
Comment (topeogunleye) [inline index.html]: - [ ] As required, please kindly ensure that all input fields have the `required` attribute to activate the validation on submit.

File: index.html [hunk 2]
```diff
@@ -193,6 +193,59 @@ <h3>Skills</h3>
...
+            </li>
+            <li>
+              <label for=""></label>
+              <textarea name="message" maxlength="500" cols="30" rows="10">
```
Comment (topeogunleye) [inline index.html]: - [ ] As required, please kindly ensure that all input fields have the `required` attribute to activate the validation on submit.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @codehass, @Talha-Hanif5666, @stevenmukama,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -193,6 +193,59 @@ <h3>Skills</h3>
...
+Enter your message here..</textarea
+              >
+            </li>
+            <li><button>Start collaboration</button></li>
```
Comment (topeogunleye) [inline index.html:236]: - [ ] [OPTIONAL] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), shown in the Figma template below:

image

the button on the pressed state has the `background: #FFFFFF;`, `color: #1A2236;`, `border-width: 2px` and on hover has the `background: #FFFFFF;`, `color: #1A2236;` `border-width: 1px`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color:  #FFFFFF;;
   color: #1A2236
   border-width: 2px;
}
```
**for hover state**
```
button:hover {
    background-color: #FFFFFF;;
    color: #1A2236
    border-width: 1px;
}

File: style.css [hunk 4]
```diff
@@ -282,4 +297,141 @@ body {
...
+    margin-top: 12px;
+    margin-bottom: 0;
+  }
+
+  .contact form ul li button:hover,
+  .contact form ul li button:focus {
+    cursor: pointer;
+    border: 1px solid #1a2236;
+    background-color: #e5e5e5;
+  }
```
Comment (codehass) [inline style.css]: Thank you for your review @topeogunleye.
I added before (button: hover) and (button: focus).
Do I change them?
Thank you.

Comment (topeogunleye) [reply style.css]: > Thank you for your review @topeogunleye. I added before (button: hover) and (button: focus). Do I change them? Thank you.

@codehass, you're welcome. `button:active` is for pressed state. I kindly suggest that you add the right colors on hover and the colors for active i.e pressed state as you have not used the right colors.

Comment (codehass) [reply style.css]: Thank you for that, I Added all these changes.

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 3 above)
Comment (Abenezer-Tilahun) [reply index.html:236]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @codehass   
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**